addons/mrp/models/mrp_workcenter.py

```diff
@@ -184,7 +184,7 @@ def unblock(self):
         if self.working_state != 'blocked':
             raise exceptions.UserError(_("It has already been unblocked."))
         times = self.env['mrp.workcenter.productivity'].search([('workcenter_id', '=', self.id), ('date_end', '=', False)])
-        times.write({'date_end': fields.Datetime.now()})
+        times.write({'date_end': fields.datetime.now()})
```

Comment:
Could you import datetime in this file instead ?. I find it easier to understand your choice